validator/server_arb/validator_spawner.go

```diff
@@ -146,6 +149,7 @@ func (v *ArbitratorSpawner) execute(
 		}
 		steps += count
 	}
+	arbitratorValidationSteps.Update(int64(steps))
```

Comment:
Instead of steps, let's use `GetStepCount()` which will be a bit more accurate, because `steps` only advances in increments of 500 million